cmd/security-secrets-setup/Dockerfile

```diff
@@ -73,20 +70,20 @@ COPY --from=build-env /edgex-go/cmd/security-secrets-setup/res/pkisetup-vault.js
 # Kong PKI/TLS materials
 COPY --from=build-env /edgex-go/cmd/security-secrets-setup/res/pkisetup-kong.json ./res
 
+# Vault's startup script
+VOLUME /vault/init
+COPY --from=build-env /edgex-go/cmd/security-secrets-setup/start_vault.sh /vault/init
+RUN chmod +x /vault/init/start_vault.sh
+
```

Comment:
Should really do some testing here to make sure that if we update start_vault.sh in our source container here, that the underling Docker volume gets the changes.  Otherwise we might want to stage this init file somewhere and have the entrypoint do a cp to /vault/init to make sure the script gets updated.

Reply:
interesting point...not sure how do we do this updating for script changes yet?

Reply:
you are right, after I tested a bit, it does suffer the updating problem.  So I am going to move them into staging area and copy the script from the entrypoint script